collector/src/cgroups.h

```diff
@@ -53,6 +57,133 @@ std::string CGroupDetected2string(CGroupDetected k);
+    { "cgroup_memory_stats_stat_pgpgout", prometheus::MetricType::Counter,
+        "Number of uncharging events to the memory cgroup" },
+    { "cgroup_memory_stats_stat_rss", prometheus::MetricType::Counter,
+        "Number of bytes of anonymous and swap cache memory (includes transparent hugepages)" },
+    { "cgroup_memory_stats_stat_rss_huge", prometheus::MetricType::Counter,
+        "Number of bytes of anonymous transparent hugepages" },
+    { "cgroup_memory_stats_stat_swap", prometheus::MetricType::Counter, "Number of bytes of swap usage" },
+    { "cgroup_memory_stats_stat_unevictable", prometheus::MetricType::Counter,
+        "Number of bytes of memory that cannot be reclaimed (mlocked etc)" },
+    { "cgroup_memory_stats_events_failcnt", prometheus::MetricType::Counter,
+        "Number of times that a usage counter hit its limit" },
+};
+
+/* structure for prometheus output : Network utilization as reported by BY-NETWORK-INTERFACE */
+static const prometheus_kpi_descriptor g_prometheus_kpi_cgroup_network[] = {
+    // cgroup : network
+    { "cgroup_network_ibytes", prometheus::MetricType::Gauge,
+        "Total number of bytes of data received by the interface" },
+    { "cgroup_network_ipackets", prometheus::MetricType::Gauge,
+        "Total number of packets of data received by the interface" },
+    { "cgroup_network_ierrs", prometheus::MetricType::Gauge,
+        "Total number of receive errors detected by the device driver" },
+    { "cgroup_network_idrop", prometheus::MetricType::Gauge, "Total number of packets dropped by the device driver" },
+    { "cgroup_network_ififo", prometheus::MetricType::Gauge, "Total number of FIFO buffer errors" },
+    { "cgroup_network_iframe", prometheus::MetricType::Gauge, "Total number of packet framing errors" },
+    { "cgroup_network_obytes", prometheus::MetricType::Gauge,
+        "Total number of bytes of data transmitted by the interface" },
+    { "cgroup_network_opackets", prometheus::MetricType::Gauge,
+        "Total number of packets of data transmitted by the interface" },
+    { "cgroup_network_oerrs", prometheus::MetricType::Gauge,
+        "Total number of transmitted errors detected by the device driver" },
+    { "cgroup_network_odrop", prometheus::MetricType::Gauge, "Total number of packets dropped by the interface" },
+    { "cgroup_network_ofifo", prometheus::MetricType::Gauge, "Total number of FIFO buffer errors" },
+    { "cgroup_network_ocolls", prometheus::MetricType::Gauge,
+        "Total number of collisions detected on the interface" },
+    { "cgroup_network_ocarrier", prometheus::MetricType::Gauge,
+        "Total number of carrier losses detected by the device driver" },
+
+};
+
+/* structure for prometheus output : PROCESS/THREAD statistics */
+static const prometheus_kpi_descriptor g_prometheus_kpi_cgroup_processes[] = {
+    // proc_loadavg
+    { "proc_loadavg_load_avg_1min", prometheus::MetricType::Counter,
```

Comment:
it's not really clear to me why proc_loadavg_load_avg_1min is listed inside 'g_prometheus_kpi_cgroup_processes'... the 1min/5min/15min avg load is read from /proc/loadavg and is related to the entire system, not to any cgroup... I think these 3 kpis need to go in system.h